Resources/Prototypes/Entities/Structures/Holographic/projections.yml

```
@@ -69,7 +69,7 @@
           layer:
             - TableLayer
     - type: TimedDespawn
-      lifetime: 180
+      lifetime: 600 # 10 Минут
```

yamllint

[warning] too few spaces before comment

(comments)